### PR TITLE
updated member profile for maintainers

### PR DESCRIPTION
**Pull request by Anita-ihuman**
Signed-off-by: Anita-ihuman <charlesanita403@gmail.com>

**Description**
I updated the member profiles of Anita Ihuman, Dhruv Patel and Vinayak Sharma .
This PR fixes # an enhancement.

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit d50bc0aee8a63e776aeee10a5ba611fd20cafe17 at: https://60f80bdf1c112b10d3ca1024--layer5ng.netlify.app

**Review by Jashpatel1 — APPROVED** [review]
Thanks @Anita-ihuman . Congratulations @dhruv0000 @vinayaksh42

**Comment by Jashpatel1** [comment]
@Anita-ihuman I guess you will have to add maintainer: yes. Sorry for giving wrong info previously.

**Comment by l5io** [comment]
🚀 Preview for commit 81a33c87bef17866b5ce3bd730a988736b469589 at: https://60f8348a4fe5e01746f8e4f6--layer5ng.netlify.app

File: src/collections/members/dhruv-patel/index.mdx [hunk 1]
```diff
@@ -1,12 +1,13 @@
 ---
 name: Dhruv Patel
-position: Contributor
+position: maintainer
```
Comment (Tanuj22) [inline src/collections/members/dhruv-patel/index.mdx:3]: Just a small nit. Can you change capitalize 'm'

File: src/collections/members/vinayak-sharma/index.mdx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 name: Vinayak Sharma
-position: Contributor
+position: maintainer
```
Comment (Tanuj22) [inline src/collections/members/vinayak-sharma/index.mdx:3]: same

**Review by Tanuj22 — CHANGES_REQUESTED** [review]

**Review by Tanuj22 — APPROVED** [review]
Thank you @Anita-ihuman

**Pull request merged**